src/test/bundle_test.ts

```diff
@@ -383,16 +365,45 @@ suite('BuildBundler', () => {
     assert.include(bundledHtml, '.simply-red', 'simple-style.css');
   });
 
-  test('bundler does not output imports which have been bundled', async() => {
+  test(
+      'bundler does not output html imports which have been bundled',
+      async() => {
+        await setupTest({
+          entrypoint: 'entrypoint-only.html',
+          sources: ['framework.html', 'entrypoint-only.html'],
```

Comment:
Are these sources necessary, given that entrypoint-only.html depends on framework.html?

Or are you adding them just to be doubly sure that framework.html was present but then was removed?